test/VoxaPlatform.spec.ts

```diff
@@ -182,12 +183,14 @@ describe("VoxaPlatform", () => {
   describe("onState", () => {
     let alexaLaunch: any;
     let dialogFlowLaunch: any;
+    let processData: any;
 
     afterEach(() => {
-      delete process.env.DEBUG;
+      process.env = processData;
```

Comment:
this was deleting my logs in the tests when running them with `DEBUG=voxa yarn test --watch --bail`